src/components/SelectBox/Select/Trigger.tsx

```diff
@@ -0,0 +1,130 @@
+  const { size, colorScheme } = useContext(SelectStyleContext)
+  const text = options.find(option => option.value === value)?.text || ''
+  const isEmpty = value === ''
+
+  const handleClick = (): void => {
+    setIsOpen?.(true)
+  }
+
+  return (
+    <StyledTriggerWrapper
+      colorScheme={colorScheme}
+      isEmpty={isEmpty}
+      size={size}
+      onClick={handleClick}>
+      <span>{text || placeholder}</span>
+      <StyledTriggerArrow
+        alt="trigger-icon"
+        boxSize="16px"
+        colorScheme={colorScheme}
+        src={ICON.CHEVRON_DOWN_16}
+      />
+    </StyledTriggerWrapper>
+  )
+}
+
+const StyledTriggerWrapper = styled.div<Omit<StyledSelectProps, 'isSelected'>>`
+  ${({ colorScheme, isEmpty, theme, size }): string => `
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+    ${applyColorScheme(colorScheme, theme)}
+    ${applySize(size, theme)}
+    color:${getFontColor({ colorScheme, isEmpty, size, theme })};
+  `}
+`
+const StyledTriggerArrow = styled(Image)<
+  Pick<StyledSelectProps, 'colorScheme'>
+>`
+  margin-left: 4px;
+  ${({ colorScheme, theme }): string => applyIconColor(colorScheme, theme)}
+`
+
+/* Styled Function */
+const applySize: ApplySize = (size, theme) => {
+  const { body02B, body02M } = theme.fonts
+
+  switch (size) {
+    case 'small':
+      return `
+        height: 32px;
+        padding: 4px 8px;
+        border-radius: 4px;
+        ${body02B}
+      `
+    case 'medium':
+      return `
+        height: 40px;
+        padding: 12px 10px;
+        border-radius: 6px;
+        ${body02M}
+      `
+  }
+}
+const applyColorScheme: ApplyColorScheme = (colorScheme, theme) => {
+  const { black, gray20, white } = theme.colors.grayScale
+
+  switch (colorScheme) {
+    case 'none':
+      return `
+        background-color: none;
+        border: none;
+      `
+    case 'light':
+      return `
+        background-color: ${white};
+        border: 1px solid ${gray20};
+      `
+    case 'dark':
+      return `
+        background-color: ${black};
+        border: none;
+      `
+  }
+}
+const getFontColor: GetFontColor = ({ isEmpty, colorScheme, size, theme }) => {
+  const { gray50, gray90, black, white } = theme.colors.grayScale
+  const isDark = colorScheme === 'dark'
+  const smallPrimary = isDark ? white : gray90
+  const mediumPrimary = isDark ? white : black
+
+  switch (size) {
+    case 'small':
+      return `${isEmpty ? gray50 : smallPrimary}`
+    case 'medium':
+      return `${isEmpty ? gray50 : mediumPrimary}`
+  }
+}
+const applyIconColor: ApplyIconColor = (colorScheme, theme) => {
+  const { gray90, white } = theme.colors.grayScale
+  const isDark = colorScheme === 'dark'
+
+  if (isDark) {
+    return `filter: ${hexToCSSFilter(white).filter}`
+  } else {
+    return `filter: ${hexToCSSFilter(gray90).filter}`
+  }
```

Comment:
삼항연산자로 변경하는 것은 어떨런지용 !?! 스타일 코드이기 때문에 간결할수록 좋을 것 같다는 생각이 들어서 의견 내봅니다!
```suggestion
    return `filter: ${hexToCSSFilter(isDark ? 'white' : 'gray90').filter}`

```

Reply:
selectBox 리팩터링은 요깃!! [08b16440](https://github.com/price-offer/offer-design-system/commit/08b16440e41486cc9ef958ed73d82e19eb810c84)